proto/v1/sql_service.proto

```diff
@@ -216,8 +216,11 @@ message Advice {
   // The advice line number in the SQL statement.
   int32 line = 5;
 
+  // The advice column number in the SQL statement.
+  int32 column = 6;
+
   // The advice detail.
-  string detail = 6;
+  string detail = 7;
```

Comment:
Maybe we shouldn't change the field numbers for any existing fields. FYI: https://protobuf.dev/programming-guides/proto3/#updating

Reply:
Right. However, this service is only used internally, so it doesn't hurt.